package.json

```diff
@@ -37,11 +38,13 @@
   },
   "devDependencies": {
     "all-contributors-cli": "3.0.6",
+    "babel-cli": "^6.18.0",
+    "babel-core": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
```

Comment:
Personally, I prefer `--save-exact`/ `-E` and to depend upon fixed dependencies' versions to reduce possible side effects that may appear with upcoming versions.